uber/http_to_https.py

```diff
@@ -1,13 +0,0 @@
-from uber.common import *
```

Comment:
+1 on removing this.

Although for future reference, we should put our comments on the Files Changed tab of the PR rather than in the commit, since the comments which @binary1230 made do not show up here due to being left on the commit itself.